python/pyiceberg/utils/datetime.py

```diff
@@ -91,6 +91,25 @@ def timestamp_to_micros(timestamp_str: str) -> int:
     raise ValueError(f"Invalid timestamp without zone: {timestamp_str} (must be ISO-8601)")
 
 
+def datetime_to_millis(dt: datetime) -> int:
```

Comment:
Could you add an unit test for this one?

Reply:
OK. added some tests